component-dev-best-practice.md

```diff
@@ -140,17 +140,17 @@ Notes for automated build tools:
 * Environment variables need to be configured in the build
 * Build tools produce badge icons that can be embedded in the Readme (should be embedded if the tool is used).
 
-##Integration application development
-###Integration application development prerequisites
+## Integration application development
+### Integration application development prerequisites
 * PS team has access to the source with all requirements
 * Meeting with Platform team representatives (for example Scrum Master) regarding E.IO platform changes (API etc.), which could potentially affect on Integration application development process
 * Access to the needed services with all needed permissions is granted
 * No open tasks that could block Integration application development process
 * Integration application architecture should be discussed and approved before starting coding.
 
-###Definition of Done (IA-DoD)
+### Definition of Done (IA-DoD)
```

Comment:
maybe `Integration application Definition of Done (CD-DoD)` instead of `Definition of Done (IA-DoD)`